README.md

```diff
@@ -18,7 +18,7 @@ We call this packaged container a _machine_.
 ### Schema
 The data that your algorithm evaluates is called its _inputs_, the data your algorithm produces is called its _outputs_.
 Both input and outputs must have well defined data type definitions called a _schema_.
-The McCoy Platform uses an Json-Schema and some custom extensions to describe the Machine inputs and outputs as well as 
+The McCoy Platform uses an YAML-Schema and some custom extensions to describe the Machine inputs and outputs as well as 
```

Comment:
You are correct that we use YAML, and that it should be clarified; however, the actual schema definition language is called JSON-Schema even if we apply it to yaml input. 

what do you think about something like this instead:
`The McCoy Platform uses [JSON-Schema](http://json-schema.org) and some custom extensions to describe the Machine inputs and outputs as well as some display information, either JSON or YAML formatted schemas are accepted.`

Reply:
sure, if it can be JSON in the label as well that would be great. The YAML is a bit uglier to algorithmically spit out and then template into a Dockerfile